ai_ref_kits/conversational_ai_chatbot/app.py

```diff
@@ -186,28 +184,28 @@ def load_speaker_embeddings(speaker_index: int) -> torch.Tensor:
     return torch.tensor(embeddings_dataset[speaker_index]["xvector"]).unsqueeze(0)
 
 
-def load_tts_model(model_dir: Path, vocoder_model_dir: Path) -> None:
+def load_tts_model() -> None:
     """
     Load text-to-speech model and assign it to a global variable
 
     Params:
         model_dir: dir with the tts model
     """
-    global ov_tts_model, ov_tts_vocoder, tts_processor, speaker_embeddings
+    global ov_tts_model, speaker_ids
 
     nltk.download('punkt_tab', quiet=True)
+    nltk.download('averaged_perceptron_tagger_eng')
 
-    tts_processor = SpeechT5Processor.from_pretrained(model_dir)
-    tts_model = SpeechT5ForTextToSpeech.from_pretrained(model_dir)
-    tts_vocoder = SpeechT5HifiGan.from_pretrained(vocoder_model_dir)
-    speaker_embeddings = load_speaker_embeddings(SPEAKER_INDEX)
+    # CPU is sufficient for real-time inference.
+    device = 'cpu' # Will automatically use GPU if available
+    model = TTS(language='EN', device=device)
+    
+    # Compile the model with OpenVINO backend for accelerated inference
+    ov_tts_model = torch.compile(model, backend='openvino')
 
-    # Compile the TTS model and vocoder with OpenVINO
-    opts = {"device": "CPU", "config":{'PERFORMANCE_HINT': "LATENCY"}}
-    ov_tts_model = torch.compile(tts_model, backend="openvino", options=opts)
-    ov_tts_vocoder = torch.compile(tts_vocoder, backend="openvino", options=opts)
+    speaker_ids = model.hps.data.spk2id
```

Comment:
do we need to assign it here? isn't it available in ` model.hps.data.spk2id` at any point?